nni/retiarii/oneshot/pytorch/supermodule/operation.py

```diff
@@ -472,6 +473,44 @@ def forward_with_args(self,
             eps,
         )
 
+class MixedLayerNorm(MixedOperation, nn.LayerNorm):
```

Comment:
Suggest adding unit tests for MixedLayerNorm: https://github.com/microsoft/nni/blob/master/test/algo/nas/test_oneshot_supermodules.py